<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="4" parallel="classes" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="4" parallel="classes" name="Test(failed)">
    <classes>
      <class name="com.crm.vtiger.contacts.CreateNewContactsTest">
        <methods>
          <include name="createContactTest"/>
          <include name="closeDbConfig"/>
          <include name="closeTheBrowser"/>
          <include name="launchBrowser"/>
          <include name="loginToAppln"/>
          <include name="dbConfig"/>
          <include name="logOut"/>
        </methods>
      </class> <!-- com.crm.vtiger.contacts.CreateNewContactsTest -->
      <class name="com.crm.vtiger.organization.CreateNewOrganizationTest">
        <methods>
          <include name="createNewOrg"/>
          <include name="closeDbConfig"/>
          <include name="closeTheBrowser"/>
          <include name="launchBrowser"/>
          <include name="loginToAppln"/>
          <include name="dbConfig"/>
          <include name="logOut"/>
        </methods>
      </class> <!-- com.crm.vtiger.organization.CreateNewOrganizationTest -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
